src/integrationTest/resources/organisation-response.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
real response from Ref-Data is a bit different:
```
{
  "organisationEntityResponse": {
    "organisationIdentifier": "0UFUG4Z",
    "name": "ia-legal-rep-org",
    "status": "ACTIVE",
    "sraId": null,
    "sraRegulated": false,
    "companyNumber": null,
    "companyUrl": null,
    "superUser": {
      "firstName": "legalrep",
      "lastName": "orgcreator",
      "email": "ia.legalrep.orgcreator@gmail.com"
    },
    "paymentAccount": [],
    "contactInformation": null
  }
}
```
there is a root node `organisationEntityResponse`

Reply:
added root node organisationEntityResponse